Classes/Neos/Neos/Ui/Domain/Model/Changes/AbstractCreate.php

```diff
@@ -20,6 +22,13 @@
     protected $nodeType;
 
     /**
+     * The referenceData for the rendered node
```

Comment:
This doesn't suffice. For inline node operations, we need to distribute fusion path for both, the reference node and the closest collection node, so the Feedback Handler knows exactly where to put the rendered node.